### DISCUSSION TITLE
Should we add proper rejections/logs for the edit function?

### DISCUSSION THREAD

**Discussion by Nico105** [Androz2091/discord-giveaways#410] [Ideas]
Currently invalid values just get ignored. + The edit function does not do anything noteworthy without options like the pause function. 
I'm not sure if that's the best option by now, since not all ignored changes can be noticed instantly cause they're not on the embed.

We either could reject them _and so the whole function_.
Or ignore them like now, but but add a log _and still execute_.

Lmk your thoughts

**Comment by DragonZV** [comment]
you could make an option called debug which enables proper rejection and logs
